### PR TITLE
feat(tier4_perception_launch): add radar launcher

### PR DESCRIPTION
**Pull request by scepter914**
Signed-off-by: scepter914 <scepter914@gmail.com>

## Description

Add radar fusion launcher.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by yukkysaito** [comment]
@scepter914 pls resolve conflict

**Comment by scepter914** [comment]
Fix conflict

**Review by yukkysaito — APPROVED** [review]
LGTM

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py [hunk 1]
```diff
@@ -75,7 +75,10 @@ def create_additional_pipeline(self, lidar_name):
                 plugin="pointcloud_preprocessor::CropBoxFilterComponent",
                 name=f"{lidar_name}_crop_box_filter",
                 remappings=[
-                    ("input", f"/sensing/lidar/{lidar_name}/outlier_filtered/pointcloud"),
+                    (
+                        "input",
+                        f"/sensing/lidar/{lidar_name}/outlier_filtered/pointcloud",
+                    ),
```
Comment (yukkysaito) [inline launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py:81]: pls revert :pray:

File: launch/tier4_perception_launch/launch/perception.launch.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="vehicle_param_file" default="$(find-pkg-share vehicle_info_util)/config/vehicle_info.param.yaml" description="path to the file of vehicle info yaml"/>
```
Comment (yukkysaito) [inline launch/tier4_perception_launch/launch/perception.launch.xml:3]: i think it is necessary

**Pull request merged**